.github/workflows/codeql.yml

```diff
@@ -0,0 +1,46 @@
+name: "Code Scanning - Action"
```

Comment:
Please make sure you're synced up to the latest main - these chanes shouldn't be in your PR